### PR TITLE
Project Exceution - k1

### PR DESCRIPTION
**Pull request by McDoxa**
I would not know why previous commits is still showing in this request.

The commit of interest is c7c2895. 

Thank you

### CODE REVIEW

File: project-javascript-k1-advanced-objects/Oluwaseyi.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                return `Mr/Ms. ${this.lastName}`;
+                } else {
+                    return `${this.firstName} ${this.lastName}`;
+                    }
+                }
+            }
+            console.log(student.fullName); 
+
+            // 5. Setter Method
+            const student = {
```
Comment (basir) [inline project-javascript-k1-advanced-objects/Oluwaseyi.html]: do not declare 2 student variable. merge it into one.

File: project-javascript-k1-advanced-objects/Oluwaseyi.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            myCourse.show();
+
+            //8. Property Validation
+            if (myCourse.teacher){
+                console.log('Teacher');
+                } else {
+                    console.log('No Teacher');
+                }
+            //9. Creating another object
+            const courseAddition = {
```
Comment (basir) [inline project-javascript-k1-advanced-objects/Oluwaseyi.html]: ` const courseAddition = Object.assign({}, myCourse, {teacher: 'Dr Mark'});`

File: project-javascript-k1-advanced-objects/Oluwaseyi.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                    }
+                }
+            };
+
+            //7. Calling the factory function
+            let myCourse = courseFactory(1,'Math', 4);
+            myCourse.show();
+
+            //8. Property Validation
+            if (myCourse.teacher){
```
Comment (basir) [inline project-javascript-k1-advanced-objects/Oluwaseyi.html]: Use `myCourse.hasOwnProperty()`

File: project-javascript-k1-advanced-objects/Oluwaseyi.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            myCourse = Object.assign(myCourse,courseAddition);
+
+            //11.
+            if (myCourse.teacher){
+                console.log('Teacher');
+                } else {
+                    console.log('No Teacher');
+                }
+
+            //12.
```
Comment (basir) [inline project-javascript-k1-advanced-objects/Oluwaseyi.html]: Use object.keys
console.log(Object.keys(myCourse).map(s=>({myCourse [s]})));

**Comment by McDoxa** [comment]
Changes effected. Thank you

**Comment by basir** [comment]
Looks good. Wait for L1

**Pull request merged**